tests/test_main.py

```diff
@@ -1,29 +1,101 @@
-from fastapi import FastAPI
+import pytest
 from fastapi.testclient import TestClient
-from gethoney.main import app
+from pydantic import ValidationError
+
+from gethoney.crud import Database
+from gethoney.main import app, get_db
 from gethoney.models import Honeypot
 
+test_db = "file::memory:?cache=shared"
+
+
+def override_db():
+    db = Database(test_db)
+    db.create_honeypot(Honeypot(name="test", url="http://12.12.1.1", description="test"))
+    yield db
+
+
+app.dependency_overrides[get_db] = override_db
+
 client = TestClient(app)
 
+#
+# Create
+#
 
-def test_read_main():
-    response = client.get("/")
-    assert response.status_code == 200
-    assert response.json() == {"Hello": "World"}
+
+def test_create_honeypot():
+    honeypot = Honeypot(name="test", url="http://12.12.1.1", description="test")
+    data = honeypot.dict()
+    response = client.post("/honeypots/", json=data)
+    assert response.status_code == 201
+    assert response.json() == data
```

Comment:
same comment here as below in regards to the intent. you should be checking that what you get is now a created resource. probably can validate it by ensuring that the resource now contains an ID.
shouldnt this test should actually be failing, since the json should have an ID returned?